http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala

```diff
@@ -760,10 +745,22 @@ class LeoPubsubMessageSubscriber[F[_]: Timer: ContextShift: Parallel](
                     None,
                     None
                   )
-                }
               }
             monitorOp = createClusterResultOpt.traverse_(createClusterResult =>
-              gkeInterp.pollCluster(PollClusterParams(clusterId, msg.project, false, createClusterResult))
+              gkeInterp
+                .pollCluster(PollClusterParams(clusterId, msg.project, false, createClusterResult))
```

Comment:
Previously `pollCluster` was throwing `HelmException` which was not caught anywhere

Reply:
is `HelmException` handled somewhere now?

Reply:
Yeah `adaptError` converts it to a `PubsubKubernetesError` which is handled.

Maybe a more general fix would be to not rethrow `RuntimeException` in this case: https://github.com/DataBiosphere/leonardo/blob/develop/http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/LeoPubsubMessageSubscriber.scala#L1080

I have another branch with more changes to GKE stuff, maybe will try that there.